web/css/root.css

```diff
@@ -121,12 +122,74 @@ nav {
   flex-direction: column;
 }
 
-nav > a {
+.header-btn {
+  min-height: 28px;
+  width: 28px;
+  cursor: pointer;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  border-radius: 0;
+  padding: 0;
+  color: var(--second-font-color);
+  background-color: var(--second-bg-color);
+  position: fixed;
+}
+
+.header-btn:not(:disabled):hover {
+  background-color: var(--second-bg-color);
+}
+
+.header-btn-arrow-left {
+  border: solid white;
+  border-width: 0 3px 3px 0;
+  display: inline-block;
+  padding: 3px;
+  width: 0px;
+  margin: auto;
+  transform: rotate(135deg);
+  -webkit-transform: rotate(135deg);
+  color: var(--second-font-color);
+}
+
+.header-btn-arrow-right {
+  border: solid white;
+  border-width: 0 3px 3px 0;
+  display: inline-block;
+  padding: 3px;
+  width: 0px;
+  margin: auto;
+  transform: rotate(-45deg);
```

Comment:
minor suggestion: can we have a common base class for both arrows so we don't need to have duplicated styles between left and right? Then the definition above could be for the `header-btn-arrow` and you would only need to overwrite the transformation for the `-right` button, not setting all styles.